### ISSUE TITLE
If username is long then it overflows out of logout button

### ISSUE THREAD

**Issue by piggyslasher** [lavab/web#891]
![image](https://cloud.githubusercontent.com/assets/1991125/8376198/f02f6e60-1c36-11e5-977a-6b8b9a1c1ca0.png)

**Comment by vLooz** [comment]
All new awesome UI is in the making but doesn't have priority right now. 

Proposal for quick fix: 
Instead of having Logout and username in one row, put the user name below Logout (left aligned) and give it a smaller and thinner font.

**Comment by kalicki** [comment]
Wow, it got weird, I don't think I need to name the "logout" you can put anywhere else?!

**Comment by vLooz** [comment]
Hi Kalicki, 
I agree with you that it's not pretty an we are going to make it much more pretty soon. 
For now we need a quick fix and I think this is the quickest ;-)

**Comment by kalicki** [comment]
And tooltip with user in hover "logout", no?

**Comment by vLooz** [comment]
For now we have labels for every of the large icons in the left panel. For consistency reasons I want to keep it this way also for Logout until we come up with a new UI design.

**Comment by piggyslasher** [comment]
Exactly. I know what to do. Will send you a screenshot tomorrow. Slack does
in nicely.

On Fri, Jun 26, 2015, 11:22 PM Felix von Looz <notifications@github.com>
wrote:

> For now we have labels for every of the large icons in the left panel. For
> consistency reasons I want to keep it this way also for Logout until we
> come up with a new UI design.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lavab/web/issues/891#issuecomment-115727739>.
>
-- 
Vik Ramanujam
vik@ramanuj.am (official)
piggyslasher@gmail.com (everything else)
+6016 656 4647

**Comment by joergG** [comment]
@piggyslasher @vLooz - Status?

**Comment by piggyslasher** [comment]
![image](https://cloud.githubusercontent.com/assets/1991125/8821585/6a08d794-3091-11e5-882d-73995f9035fb.png)
moving to testing

**Comment by joergG** [comment]
looks good...

**Comment by joergG** [comment]
@piggyslasher - like discussed, we found it to be buggy in IE. hence reopening it  :-)

![username](https://cloud.githubusercontent.com/assets/12829397/8873360/1bb92000-3206-11e5-8320-bece22d91235.jpg)
